src/shared/lib/MutationUtils.ts

```diff
@@ -365,3 +365,24 @@ export function isValidGenomicLocation(mutation: Partial<Mutation>): boolean {
 
     return false;
 }
+
+export function extractAnnotatedMutation(
```

Comment:
this is more like `finding` than `extracting`, so maybe
```suggestion
export function findMatchingAnnotatedMutation(
```